### DISCUSSION TITLE
Chinese month names in Gantt, en-US culture

### DISCUSSION THREAD

**Discussion by cetinbas** [dotnetcore/BootstrapBlazor#3111] [Q&A] [answered]
I am seeing month names in Chinese in Gantt. I am using en-US culture. Am I missing something because in other components dates are formatted in English?

**Comment by ArgoZhang** [comment]
This component is provided by the community and may not have been localized. Can you submit a PR to fix this issue?

**Comment by ArgoZhang** [comment, marked as answer]
link issue #3114
